virtual_rainforest/models/hydrology/constants.py

```diff
@@ -90,8 +90,8 @@ class HydroConsts:
     """Parameter used to estimate vegetation density for maximum canopy interception
     capacity estimate after :cite:t:`von_hoyningen-huene_interzeption_1981`."""
 
-    groundwater_capacity: float = 0.9
-    """Ground water storage capacity in relative volumetric water content. This might be
+    groundwater_capacity: float = 500
+    """Ground water storage capacity in millimeters. This might be
     replaced with the implementation of below ground horizontal flow."""
```

Comment:
Minor nit, but I'd generally have only a single sentence on the first line of a docstring, then an empty line then the rest of the description. `flake8-docstring` enforces this for function docstrings, but seemingly not for variables.